src/src/Helpers/Hardware.cpp

```diff
@@ -622,10 +622,15 @@ uint8_t temprature_sens_read();
 #endif  // ESP32
 
 float getInternalTemperature() {
```

Comment:
Why isn't this entire function wrapped in `#ifdef ESP32` checks?
No need for having the function declaration on ESP8266.

Reply:
That's guarded by the check on `FEATURE_INTERNAL_TEMPERATURE`, that'll be only enabled for ESP32. I can add an extra check it's not 'accidentally' enabled for ESP8266.